tests/README.md

```diff
@@ -111,24 +110,29 @@ go test -v -timeout 1800s -run SmokeSuite github.com/rook/rook/tests/integration
 
 ##### To run specific without installing rook
 ```
-go test -v -timeout 1800s -run SmokeSuite github.com/rook/rook/tests/integration --skip_install_rook=true
+go test -v -timeout 1800s -run SmokeSuite github.com/rook/rook/tests/integration --skip_install_rook
 ```
 If the `skip_install_rook` flag is set to true, then rook is not uninstalled either. 
 
 #### Run Longhaul Tests
-Using the `go test -count n` option, any tests can be repeated `n` number of times to simulate load or longhaul test. Although 
-any test can be converted to a longhaul test, it's ideal to run integration tests under load for an extended period. Also a new custom test flag `load_parallel_runs` is added to control the number of concurrent operations being performed.
-For example look at the [block long haul test](/tests/block/k8s/longhaul/basicBlockonghaul_test.go)
- 
- To run a longhaul test you can run any integration test with `-count` and `--load_parallel_runs` options
- e.g.
+[longhaul](/tests/block/k8s/longhaul) Tests are integration tests that run for extended period of time. A load profile can be configured
+using the following load test flags 
++ `load_parallel_runs`: controls number of concurrent operations reading and writing data
```

Comment:
You list these flags here and in the table above? I would suggest removing them from the table above. Then it will keep the main tests simple, and people running load tests can look here for other flags. And maybe you could create a new table here instead of the bulleted list.

Reply:
done.